proto/lbm/zkauth/v1/zkauth.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+package lbm.zkauth.v1;
+
+import "cosmos_proto/cosmos.proto";
+import "cosmos/crypto/secp256k1/keys.proto";
+import "cosmos/auth/v1beta1/auth.proto";
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/Finschia/finschia-sdk/x/zkauth/types";
+
+// ZKAuthAccount is an Account generated by ZKAuth. 
+// It conforms to the BaseAccount interface, but pubkey is not present and returns nil.
+message ZKAuthAccount {
+    option (gogoproto.goproto_getters)         = false;
+    option (gogoproto.equal)                   = false;
+    option (cosmos_proto.implements_interface) = "cosmos.auth.v1beta1.AccountI";
+  
+    string address        = 1;
+    uint64 account_number = 2;
+    uint64 sequence       = 3;
+}
+
+// ZKAuthInputs is a parameter required for zkp verify. 
+// Contains the values needed to calculate "all inputs hash".
+message ZKAuthInputs {
+    bytes proof_points = 1;
+    Claim iss_base64_details = 2;
```

Comment:
Our model does not need `Claim` structure. So please use iss_base64 string or byte directly.

Reply:
After all, is it base64 instead of iss_f? At the meeting, I thought it was decided on iss_f.

Reply:
Now it's ok. But we need to discuss more about the name and type. Let's discus it later.